src/environmentd/tests/sql.rs

```diff
@@ -1309,6 +1313,62 @@ fn test_timestamp_recovery() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+#[test]
+fn test_timeline_read_holds() -> Result<(), Box<dyn Error>> {
```

Comment:
On `main` this hangs forever (for some reason the `mz_ore::test::timeout` wasn't actually timing out.